src/Microsoft.Graph.Core/Requests/BaseFunctionMethodRequestBuilder.cs

```diff
@@ -121,5 +116,17 @@ private string EscapeStringValue(string value)
             // Per OData spec, single quotes within a string must be escaped with a second single quote.
             return value.Replace("'", "''");
         }
+
+        /// <summary>
+        /// Set the parameter string on the request URL. MUST be used after all 
+        /// parameters have been set in the generated request builder. Used for
+        /// each section. Adds an empty () if no parameters are present.
+        /// </summary>
+        protected void SetParameterString()
```

Comment:
Can a different name be used for this method? Example - > BuildRequestURL() or something not similar to `SetParameter()`.

Reply:
Some name that indicates the concatenation/ building of the complete URL.

Reply:
This won't build the request URL. It only adds the current request builders parameters to the request URL. The request URL can have query strings added later at the end of the request builder chain. 

I'm comfortable with them having similar names as the functionality is related. Would SetParameterStringOnUrl() be clearer?